cni/network/invoker_azure.go

```diff
@@ -29,7 +29,7 @@ const (
 
 type AzureIPAMInvoker struct {
 	plugin delegatePlugin
-	nwInfo *network.NetworkInfo
+	nwInfo *network.EndpointInfo
```

Comment:
When I merged the network info struct into the endpoint info struct, some fields had the same names, but I was not sure if they should be kept separate. For example, "DNS"-- previously we would have two separate fields: nwInfo.DNS and epInfo.DNS-- there may have been a difference between the DNS in nwinfo and epinfo. I kept the variable names as nwInfo or epInfo so that if there were another instance where I had accidentally merged two fields together when they should not have been, I would be able to tell if that field was accessed from a network info or endpoint info previously. Because of this, I want to keep those names as they are for now at least.

Reply:
I want to keep the variable names as is because if we find out later that I accidentally merged something I should not have, it will be incredibly difficult to pin-point which endpointinfos were previously used in the context of networks, and which ones were used in the context of endpoints